src/lib/analytics.js

```diff
@@ -172,6 +172,38 @@ function getNavigationType() {
   return '(not set)';
 }
 
+function getPrerenderRules() {
+  return [...document.querySelectorAll('script[type=speculationrules]')]
+    .map((s) => {
+      try {
+        return JSON.parse(s.textContent).prerender;
+      } catch {
+        // Ignore parse errors.
+      }
+    })
+    .flat() // Remove scripts with errors or no prerender rules.
+    .filter((rule) => rule && rule.source === 'list');
+}
+
+function logPrerenders() {
+  const prerenderURLs = new Set(
+    getPrerenderRules()
+      .map((r) => r.urls)
+      .flat(),
+  );
+
+  prerenderURLs.forEach((prerenderURL) => {
+    ga('send', 'event', {
+      eventCategory: 'Site-Wide Custom Events',
+      eventAction: 'Prerender attempt',
```

Comment:
```suggestion
      eventAction: 'prerender_attempt',
```

Eventually this is going to switch to GA4, so I think it makes sense to align to the conventions that system uses. See: https://support.google.com/analytics/answer/9234069